harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -28,15 +29,195 @@
     pass
 
 
-class PyTorchTrialController(det.TrialController):
-    def __init__(self, trial_inst: det.Trial, *args: Any, **kwargs: Any) -> None:
-        super().__init__(*args, **kwargs)
+class TrainUnit:
```

Comment:
Can you put at least 1-sentence docstrings on these new classes you created?  For other devs, basically.